crossbeam-utils/Cargo.toml

```diff
@@ -24,6 +24,7 @@ alloc = []
 [dependencies]
 cfg-if = "0.1.2"
 lazy_static = { version = "1.1.0", optional = true }
+arr_macro = "0.1.3"
```

Comment:
As this is a proc-macro, it may greatly increase the compile time. cc @stjepang

Reply:
I removed the dependency and manually expanded `arr![SeqLock::init(), 97]`.